roles/pgbouncer/templates/pgbouncer.ini.j2

```diff
@@ -1,9 +1,12 @@
 [databases]
+{% if pgbouncer_auth_user | bool %}
+* = host=127.0.0.1 port={{ postgresql_port }} dbname=postgres auth_user={{ pgbouncer_auth_username }}
```

Comment:
You have specified the "auth_user" option globally, is it really necessary to duplicate this option for each pool?

I think it's not necessary.

Reply:
You're right. I'll remove the option from the pool